lib/index.js

```diff
@@ -76,13 +76,16 @@ export default class PhoneInput extends Component {
       key: index,
       image: Flags.get(country.iso2),
       label: country.name,
-      dialCode: `+${country.dialCode}`,
+      dialCode: `+${(country && country.dialCode) || ''}`,
```

Comment:
I'm not convinced this line change is necessary. The countries collection should only contain objects. If `country` is a null-type it's going to crash on the line above anyway.